synchronizer/synchronizer.go

```diff
@@ -339,31 +371,54 @@ func (s *Synchronizer) AdvanceView(syncInfo hotstuff.SyncInfo) {
 
 	leader := s.leaderRotation.GetLeader(s.currentView)
 	if leader == s.opts.ID() {
-		s.consensus.Propose(syncInfo)
+		s.consensus.Propose(syncInfo.WithQC(s.highQC))
 	} else if replica, ok := s.configuration.Replica(leader); ok {
 		replica.NewView(syncInfo)
 	}
 }
 
+// isInPipelineStretch checks wether the given view lies
+// less then the pipelinestretch from the highQC
+func (s *Synchronizer) isInPipelineStretch(v hotstuff.View) bool {
```

Comment:
What about just `inPipeline` or `isInPipeline` or `inPipelineRange`?

Typo in comment: wether -> whether. But perhaps this is more clear?:
```go
// inPipeline returns true if the given view is in pipeline range from highQC.
```

Reply:
fixed in [df3fc32](https://github.com/relab/hotstuff/pull/78/commits/df3fc324538afeeadc39d8adedcddb7d618f4ee8)
```Go
// inPipeline checks if the given view is in pipeline range from highQC.
func (s *Synchronizer) inPipeline(v hotstuff.View) bool {
```